R/quantile_cens.R

```diff
@@ -1,32 +1,62 @@
 #' Calculate quantiles respecting the censored data
 #' 
-#' @param x data
-#' @param p quantile
-#' @param limit censoring limit
-#' @param cens censoring direction (left/right)
-#' 
+#' @inheritParams stats::quantile
+#' @inheritParams loq_frac
+#' @return The quantile of \code{x} treating \code{NA} values as censored
 #' @export
-quantile_cens <- function(x, p = 0.5, limit = 1, cens = "left") {
-  if(cens %in% c("left", "lower", "bloq", "loq", "lloq")) {
+quantile_cens <- function(x, probs = 0.5, limit = 1, cens = c("left", "right", "neither", "both")) {
+  cens <- match.arg(cens)
+  if (cens == "left") {
     x[is.na(x)] <- -Inf
     x[x<limit] <- -Inf
-  } else {
+  } else if (cens == "right") {
     x[is.na(x)] <- Inf
     x[x>limit] <- Inf
+  } else if (cens == "neither") {
+    # TODO: Review on 2021-04: quantile_cens can now just work like quantile
+    # (simplifying other code that checks if quantile_cens() or quantile()
+    # should be used).  To consider: Should NA values be considered missing or
+    # an error?
```

Comment:
That sounds reasonable to me.  I will implement that leaving in the "to consider" comment.  The reason for keeping the "to consider" comment is that In some places in the code, censored data are converted to `NA`.  I'll open a separate discussion issue for that since it is a broader discussion than this PR covers.

Reply:
Ref #80